packages/manager/modules/pci/src/projects/project/storages/containers/containers.service.js

```diff
@@ -477,22 +479,29 @@ export default class PciStoragesContainersService {
       });
   }
 
-  deleteHighPerfObject(projectId, containerId, objectKey, containerRegion) {
+  deleteHighPerfObject(
```

Comment:
**nitpick:** Since this function performs delete of both `highPerf` as well as 's3`. It is better to rename the function

Reply:
Okay Anoop